### ISSUE TITLE
Any chance updating pydantic dependency?

### ISSUE THREAD

**Issue by davidban77** [nornir-automation/nornir#428]
Hello!

Looking at your contrib guidelines I was wondering when do you have plans to update the dependencies?

I am currently working with another project [gns3fy](https://github.com/davidban77/gns3fy). And have a pydantic version conflict.

Currently is on v0.32

https://pydantic-docs.helpmanual.io/#v0-32-2-2019-08-17.

Thanks!

**Comment by dmfigol** [comment]
I was wondering when something like this will come up - people using other libraries together with nornir which have dependencies incompatible with ours. The problem with pydantic specifically is that there is no major release which we could pin to and with nornir we pin major ones (or specific version until 1.0) 
Going back to your specific question. We would appreciate if you could check if the tests are still working (and potentially fix broken ones) on that version of pydantic and submit PR.

**Comment by davidban77** [comment]
Forked it, did the change and started testing. It seems like there is an issue with `sphinx` testing and Jinja2.
```
Creating network "nornir_net1" with driver "bridge"
Creating nornir_dev2.group_1_1  ... done
Creating nornir_dev1.group_1_1  ... done
Creating nornir_dev4.group_2_1  ... done
Creating nornir_dev5.no_group_1 ... done
Creating nornir_dev3.group_2_1  ... done
Creating nornir_httpbin_1       ... done
PYTHON=3.6 docker-compose -f docker-compose.yaml \
		run --rm nornir black --check nornir tests docs
All done! ✨ 🍰 ✨
108 files would be left unchanged.
# TODO REPLACE with: sphinx-build -n -E -q -N -b dummy -d docs/_build/doctrees docs asd
PYTHON=3.6 docker-compose -f docker-compose.yaml \
		run --rm nornir sphinx-build -W -b html -d docs/_build/doctrees docs docs/_build/html
Running Sphinx v1.8.5

Exception occurred:
  File "/usr/local/lib/python3.6/site-packages/jinja2/environment.py", line 430, in getattr
    return getattr(obj, attribute)
jinja2.exceptions.UndefinedError: 'dict object' has no attribute 'properties'
The full traceback has been saved in /tmp/sphinx-err-z7m1zrpa.log, if you want to report the issue to the developers.
```
Will take a deeper look at it tomorrow, calling it a day.

Thanks!

**Comment by dmfigol** [comment]
Fixed with #449
